owiddata/generate-owiddata-stats.sh

```diff
@@ -3,18 +3,20 @@
 # Specify the relative path to the file in the repo
 OWID_REPO_PATH=public%2Fdata%2Fvaccinations%2Fvaccinations.json
 
-
 # Get the most recent commit affecting this file and store the commit sha and date
 # See https://stackoverflow.com/questions/50194241/get-when-the-file-was-last-updated-from-a-github-repository/50204589
 OWID_COMMIT_JSON=$(curl -sS "https://api.github.com/repos/owid/covid-19-data/commits?path=$OWID_REPO_PATH&per_page=1")
 export OWID_COMMIT_SHA=$(echo $OWID_COMMIT_JSON | python -c "import sys, json; print(json.load(sys.stdin)[0]['sha'])")
 export OWID_COMMIT_DATE=$(echo $OWID_COMMIT_JSON | python -c "import sys, json; print(json.load(sys.stdin)[0]['commit']['author']['date'])")
 
+# Input file(s)
+VACCINE_PLATFORMS=owiddata/vaccine_platforms.csv
```

Comment:
Where does this csv come from?  Is it manually created once?  Will it ever need to be updated?

Reply:
We will need to update it :( I haven't found a good external resource to unify these. When new vaccines are added in OWID, this will exit with an error saying to update the CSV. (Line 109 in the python script)

Reply:
That approach seems good enough for now.  We may not notice if these vaccine figures stop updating because of that error, but we can check periodically.